Spartan/spartan.download.recipe

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>Description</key>
+	<string>Downloads the latest version of Spartan from www.wavefun.com</string>
+	<key>Identifier</key>
+	<string>com.github.its-unibas.download.spartan</string>
+	<key>Input</key>
+	<dict>
+		<key>NAME</key>
+		<string>Spartan</string>
+		<key>SEARCH_PATTERN</key>
+		<string>"(http:\/\/downloads\.wavefun\.com\/Mac\/Spartan..V...\.dmg)".*Macintosh.*v\..\..\..</string>
+		<key>SEARCH_URL</key>
+		<string>https://www.wavefun.com/downloads</string>
+	</dict>
+	<key>MiniumumVersion</key>
+	<string>1.0</string>
+	<key>Process</key>
+	<array>
+		<dict>
+			<key>Arguments</key>
+			<dict>
+				<key>re_pattern</key>
+				<string>%SEARCH_PATTERN%</string>
+				<key>url</key>
+				<string>%SEARCH_URL%</string>
+			</dict>
+			<key>Processor</key>
+			<string>URLTextSearcher</string>
+		</dict>
+		<dict>
+			<key>Arguments</key>
+			<dict>
+				<key>url</key>
+				<string>%match%</string>
+				<key>filename</key>
+				<string>%NAME%.dmg</string>
+			</dict>
+			<key>Processor</key>
+			<string>URLDownloader</string>
+		</dict>
+		<dict>
+			<key>Processor</key>
+			<string>EndOfCheckPhase</string>
+		</dict>
+		<dict>
+			<key>Arguments</key>
+			<dict>
+				<key>dmg_path</key>
+				<string>%pathname%</string>
+			</dict>
+			<key>Processor</key>
+			<string>AppDmgVersioner</string>
```

Comment:
Where is this information used? As far as I can tell, `version` is later overwritten by the `Versioner` processor.

Reply:
`AppDmgVersioner` provides the following things from a DMG: 
- `app_name` the name of the Application in the DMG used for the `CodeSignatureVerifier`
- `bundleid` the Bundle identifier of the app used for the `CodeSignatureVerifier`
- `version` which in this case is the CFBundleShortVersionString. Using CFBundleVersion as version and CFBundleShortVersionString, I thought it would be better to label them that way. That is the reason why I overwrote the version.